homeassistant/components/sensor/gtfs.py

```diff
@@ -189,22 +202,28 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         pygtfs.append_feed(gtfs, os.path.join(gtfs_dir, data))
 
     add_entities([
-        GTFSDepartureSensor(gtfs, name, origin, destination, offset)])
+        GTFSDepartureSensor(pygtfs=gtfs,
+                            name=name,
+                            origin=origin,
+                            destination=destination,
+                            offset=offset,
+                            mode=mode)])
 
 
 class GTFSDepartureSensor(Entity):
     """Implementation of an GTFS departures sensor."""
 
-    def __init__(self, pygtfs, name, origin, destination, offset):
+    def __init__(self, pygtfs, name, origin, destination, offset, mode):
         """Initialize the sensor."""
         self._pygtfs = pygtfs
         self.origin = origin
         self.destination = destination
         self._offset = offset
         self._custom_name = name
         self._icon = ICON
+        self._mode = mode
         self._name = ''
-        self._unit_of_measurement = 'min'
+        self._unit_of_measurement = UNIT_OF_MEASUREMENT
```

Comment:
Opportunistic cleanup by transforming this in-line string into a component variable.

Reply:
No longer relevant.